AdminCore.DatabaseMigration/db/migrations/V1.0__original_schema.sql

```diff
@@ -113,7 +113,12 @@ CREATE TABLE IF NOT EXISTS public.event_type
 (
     event_type_id        integer,
     description character varying(255) COLLATE pg_catalog."default",
-    CONSTRAINT event_type_pkey PRIMARY KEY (event_type_id)
+		employee_role_id integer NOT NULL,
```

Comment:
This will change the checksum of the migration meaning we have to clear down the DB etc. I think it's fine but it's something to be aware of for the next PR.